kickstarts/partials/post/ui_compile.ks.erb

```diff
@@ -1,4 +1,4 @@
-npm install -g gulp bower
+npm install -g gulp bower yarn
 
 pushd /var/www/miq/vmdb
   bower -F --allow-root install
```

Comment:
@simaishi I'll take care of removing them once Ops UI is no longer using bower :) For now, they are still needed